web-compat/web-compat-store/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You don't need a manifest in this module. You can delete and add the namespace in the build.gradle file